src/libs/AxelarQueryAPI.ts

```diff
@@ -80,20 +76,16 @@ export class AxelarQueryAPI {
     assetDenom: string,
     amountInDenom: number
   ): Promise<TransferFeeResponse> {
-    try {
-      if (!this.axelarQueryClient)
-        this.axelarQueryClient = await AxelarQueryClient.initOrGetAxelarQueryClient({
-          environment: this.environment,
-          axelarRpcUrl: this.axelarRpcUrl,
-        });
-      return await this.axelarQueryClient.nexus.TransferFee({
-        sourceChain: sourceChainName,
-        destinationChain: destinationChainName,
-        amount: `${amountInDenom.toString()}${assetDenom}`,
+    if (!this.axelarQueryClient)
+      this.axelarQueryClient = await AxelarQueryClient.initOrGetAxelarQueryClient({
+        environment: this.environment,
+        axelarRpcUrl: this.axelarRpcUrl,
       });
-    } catch (e: any) {
-      throw e;
-    }
+    return await this.axelarQueryClient.nexus.TransferFee({
```

Comment:
same

Reply:
replace all 17 occurrences of `return await` https://github.com/axelarnetwork/axelarjs-sdk/pull/193/commits/8ec59dd55e97caf6259052f82a914601c140d013